### ISSUE TITLE
merge script - obsolete package not copied

### ISSUE THREAD

**Issue by DaJansenGit** [lbl-srg/BuildingsPy#294]
With https://github.com/RWTH-EBC/AixLib/issues/740 we just merged the current version of IBPSA lib into the AixLib. We noticed that with the current version of the merge script the new obsolete package is not merged into AixLib. Is this intentional?

**Comment by mwetter** [comment]
Yes, this is intentional as models in the `Obsolete` packages are scheduled for removal (and maybe already removed in some downstream libraries).
